impeller/docs/faq.md

```diff
@@ -100,3 +100,113 @@
+    are [discussed in this
+    tech-talk](https://www.youtube.com/watch?v=gKrYWC_SDxQ). The renderer is way
+    more advanced today but that talk describes the very early thinking,
+    motivations, and decisions.
+  * With Impeller shipping on multiple apps, the open issues related to shader
+    compilation jank have been fixed and closed.
+  * Today, Impeller outperforms the old renderer not only in worst-frame timed
+    benchmarks, but is also faster on average.
+* What makes Impeller different, compared to Skia, for Flutter?
+  * All of Impellers shaders are [authored and
+    compiled](https://github.com/flutter/engine/tree/0a8de3dd3285c0b64de47630a8218ae38b8e04e1/impeller#the-offline-shader-compilation-pipeline)
+    during build time and packaged with the Flutter engine. There is no runtime
+    shader generation, reflection, or compilation. Skia can and does generate
+    and compile shaders at runtime.
+  * By necessity, Impeller has a bounded set of shaders that are known ahead of
+    time. Due to the way rendering intent is parameterized, Impeller also needs
+    way fewer shaders than Skia. All the graphics pipelines needed by Impeller
+    are ready well before the Flutter applications Dart isolate is launched.
+    Skia can generate and compile shaders during frame workloads leading to
+    worse worst-frame times.
+  * Because a large chunk of Impeller works at build time, Impellers binary size
+    impact is a fraction as that of Skias. Since there is no shader compilation
+    and reflection machinery at runtime, Impeller [adds around
+    100kb](https://github.com/flutter/flutter/issues/123741#issuecomment-1553382501)
+    of binary size at runtime (compressed) even when all the generated shaders
+    are packaged in a Flutter engine. Removing Skia GPU (which includes the SKSL
+    shader compilation machinery) [reduces the binary size of the Flutter engine
+    by 17%](https://github.com/flutter/engine/pull/52748). That is to say, the
+    machinery to generate, parse, compile, and reflect shaders is larger than
+    the backend specific shaders themselves. Using an Impeller only build leads
+    to much smaller Flutter engines.
+  * Impeller doesn't have a software backend unlike Skia. Instead, software
+    rendering using Impeller can be achieved using a Vulkan or OpenGL
+    implementation running in software using projects like SwiftShader,
+    LLVMPipe, etc...
+* Why is Impeller called "Impeller"?
+  * Impeller began as a modest experiment to solve the problem of shader
+    compilation jank. After the theory of operation was settled on, the process
+    of hacking together a prototype began with a component tacked onto Flutters
+    compositor. The compositor happens to be called "flow". Since impellers
+    modify fluid flow, the name seemed apt. Its also an internal component and
+    perhaps 30 seconds were spent thinking of a name. Conflicts were discovered
```

Comment:
remove the sentence starting with Conflicts

Reply:
Done.